Packages/com.unity.multiplayer.samples.coop/Utilities/SceneManagement/SceneLoaderWrapper.cs

```diff
@@ -69,7 +69,7 @@ public void AddOnSceneEventCallback()
         /// <param name="loadSceneMode">If LoadSceneMode.Single then all current Scenes will be unloaded before loading.</param>
         public void LoadScene(string sceneName, LoadSceneMode loadSceneMode = LoadSceneMode.Single)
         {
-            if (m_NetworkManager != null && m_NetworkManager.IsListening && m_NetworkManager.NetworkConfig.EnableSceneManagement)
+            if (m_NetworkManager != null && m_NetworkManager.IsListening && !m_NetworkManager.ShutdownInProgress && m_NetworkManager.NetworkConfig.EnableSceneManagement)
```

Comment:
This was needed because else when calling Shutdown on NetworkManager and after call LoadScene nothing would happen. Because Shutdown is asynchronous. Changing this did not seem to break anything but it feels a bit sketchy

Reply:
Should IsListening return false when ShutdownInProgress is true?